### ISSUE TITLE
using number* as input for expand-lst 

### ISSUE THREAD

**Issue by jlquispec** [cac-t-u-s/om-sharp-users#40]
first way: #: appears 

![image](https://user-images.githubusercontent.com/53123770/127379303-9fffb598-5a1c-4d29-aae4-3e0f25c58ef6.png)

![image](https://user-images.githubusercontent.com/53123770/127379328-01a5a642-8e6f-47c8-9d9b-fa6978e14194.png)

![image](https://user-images.githubusercontent.com/53123770/127379348-a7fc461b-bccf-42fb-abf6-31d6ae77d62c.png)

second way: works

![image](https://user-images.githubusercontent.com/53123770/127379396-936dbe96-f3ba-4975-8a05-452e922c7460.png)
![image](https://user-images.githubusercontent.com/53123770/127379413-9e995a33-d865-4c8d-8255-10744a7fe3ff.png)

![image](https://user-images.githubusercontent.com/53123770/127379434-488d5f2b-76c3-475d-be1b-74f06a59527a.png)

**Comment by j-bresson** [comment]
Hi @jlquispec sorry I can't really understand here. I think I would need more details (or less!:)

What I can see and explain, is that you can not create a box from "8*", because it is a symbol (a [Lisp symbol](https://www.cs.cmu.edu/Groups/AI/html/cltl/clm/node27.html)), and the "box-creation mechanism" will try to evaluate this symbol to see if it is bound to a function or a class, in order to create a box corresponding to this function or class. 

If you want to create a simple "[value box](https://cac-t-u-s.github.io/pages/value-box#symbols)" with it, you need to use a quote (`'`) to specify that this symbol should not be evaluated (similar to the use of [quote with a Lisp interpreter](http://clhs.lisp.se/Body/s_quote.htm)!)

In other words, you can type `'8*` to create a simple value box containing the symbol `8*`.

**Comment by jlquispec** [comment]
Thanks Jean!

> If you want to create a simple "value box" with it, you need to use a quote (') to specify that this symbol should not be evaluated (similar to the use of quote with a Lisp interpreter!)

That works! Excuse me for this very basic question!
